www/src/privacy.md

```diff
@@ -51,11 +49,11 @@ Responsible for the processing of your personal data is:
 E-mail: {{ config.privacy_mail }}
 
 We have appointed a data protection officer:
-
 ```
 {{ config.privacy_pdo }}
 ```
 
+
```

Comment:
same with changing blank lines. The markdown rendering equalizes them anyway; the merge conflicts are not worth it.